src/lib.rs

```diff
@@ -6,11 +6,18 @@ use std::iter::FromIterator;
 use std::fmt::{Debug, Formatter, Result as FmtResult};
 use std::cmp;
 
+/// An element of an interval tree.
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
`derive(Debug)` requires `K` and `V` both to be Debug. What do you think about manually writing the debug code with `impl<K, V> Debug for Element<K, V> where K: Debug + V: Debug`?